extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -1035,7 +1035,7 @@ function formatValue( $field, $value ) {
 		global $wgWikiaBaseDomain;
 		switch ($field) {
 			case "city_url":
-				preg_match("/http:\/\/([\w\.\-]+)\//", $value, $matches );
+				preg_match("/https?:\/\/([\w\.\-]+)\//", $value, $matches );
 				$sRetval = str_ireplace(".".$wgWikiaBaseDomain, "", $matches[1]);
 				return $sRetval;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the code after this "return". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1763)

[project-id]: wikia:app